### PR TITLE
[Web-37] - signin

### PR DESCRIPTION
**Pull request by arsalan96**
- Make a common folder for generic components and create a generic button component

- UI of sign in page is completed
- Inline styling fixed

### CODE REVIEW

File: src/pages/login.tsx [hunk 1]
```diff
@@ -12,17 +12,27 @@ import {
...
 } from '@mui/material';
 import googleIcon from '../assets/images/google-icon.svg';
 import microsoftIcon from '../assets/images/microsoft-icon.svg';
-import loginBackground from '../assets/images/login-background.png';
+import loginBackground from '../assets/images/auth_left_illustration.svg';
+import whiskLogo from '../assets/images/whisk-logo.svg';
 import { signIn } from 'next-auth/react';
 import { LoginInput } from '@types';
 import { useRouter } from 'next/router';
+import { GenericButton } from '@components/common/Button';
```
Comment (UmairZahid490) [inline src/pages/login.tsx:26]: create an index.ts file to export the components as done in other folders

**Review by UmairZahid490 — CHANGES_REQUESTED** [review]

File: src/pages/login.tsx [hunk 2]
```diff
@@ -12,17 +12,27 @@ import {
...
 
 const classes = {
-	signInContainer: {
+	signInContainer1: {
+		padding: '0px 120px',
+		height: '100%',
+		flexDirection: 'column',
+		justifyContent: 'center'
+	},
+	signInContainer2: {
```
[the remark this answers is not in the document]
Comment (arsalan96) [reply src/pages/login.tsx:35]: Done

File: src/pages/login.tsx [hunk 3]
```diff
@@ -125,29 +153,21 @@ const Login: NextPage = () => {
...
-								top: 0,
-								bottom: 0,
-								right: 0,
-								left: 0,
-								backgroundImage: `url(${loginBackground.src})`,
-								backgroundRepeat: 'no-repeat',
-								backgroundSize: 'cover'
-							}}
-						></Grid>
+						<Grid sx={classes.leftLoginBackground} item xs={12}></Grid>
```
[the remark this answers is not in the document]
Comment (arsalan96) [reply src/pages/login.tsx:159]: Done

File: src/pages/login.tsx [hunk 4]
```diff
@@ -125,29 +153,21 @@ const Login: NextPage = () => {
...
 					</Hidden>
 				</Grid>
 			</Grid>
-			<Grid item xs={12} sm={4}>
-				<Grid container sx={classes.signInContainer}>
-					<Typography variant='h4'>Sign in</Typography>
+			<Grid item xs={12} sm={6}>
+				<Grid container sx={isSmallScreen ? classes.signInContainer2 : classes.signInContainer1}>
+					<Grid item>
+						<img src={whiskLogo.src} width='100%' height='100%' />
```
[the remark this answers is not in the document]
Comment (arsalan96) [reply src/pages/login.tsx:166]: Done

**Review by UmairZahid490 — APPROVED** [review]
Approving

**Pull request merged**